src/main/webapp/app/complaints/complaints-for-tutor/complaints-for-tutor.component.ts

```diff
@@ -36,7 +36,9 @@ export class ComplaintsForTutorComponent implements OnInit {
             });
         }
     }
-
+    /** Allows user to respond to a complaint or a more feedback request if they have the required rights
+     * @param acceptComplaint - flag that tells if a complaint is accepted or not
```

Comment:
I would prefer to use the style described here:
https://jsdoc.app/about-getting-started.html

So either a one line comment OR a multi-line comment where the first and last line only contain the comment start/end stars.

This also applies to other occurrences.